packages/stats/gbstats/bayesian/tests.py

```diff
@@ -313,3 +351,121 @@ def compute_result(self) -> BayesianTestResult:
                 result, self.traffic_proportion_b, self.phase_length_days
             )
         return result
+
+
+class GaussianEffectABTest(GaussianBayesianABTest):
+    def __init__(
+        self,
+        stat_a: Union[
+            SampleMeanStatistic,
+            RatioStatistic,
+            QuantileStatistic,
+            QuantileStatisticClustered,
+        ],
+        stat_b: Union[
+            SampleMeanStatistic,
+            RatioStatistic,
+            QuantileStatistic,
+            QuantileStatisticClustered,
+        ],
+        config: Union[EffectConfigFlatPrior, EffectConfigStandardPrior],
+    ):
+        super().__init__(stat_a, stat_b, config)
+        self.prior_effect = config.prior_effect
+        self.stat_a = stat_a
+        self.stat_b = stat_b
+
+    def compute_result(self):
+        self.compute_frequentist_moments()
+        self.compute_bayesian_moments()
+        risk = [self.risk]
+        ci = self.credible_interval(
+            self.mean_diff, self.std_diff, self.ccr, self.relative
+        )
+        ctw = self.chance_to_win(self.mean_diff, self.std_diff)
+        if self.relative:
+            if self.stat_a.mean == 0.0:
+                raise ValueError(
+                    "Cannot compute relative effect when control mean is 0."
+                )
+            else:
+                self.mean_diff /= np.abs(self.stat_a.mean)
+                self.std_diff /= np.abs(self.stat_a.mean)
+        # probably better to tear these out of superclass, have this be standalone class
+        result = BayesianTestResult(
+            chance_to_win=ctw,
+            expected=self.mean_diff,
+            ci=ci,
+            uplift=Uplift(
+                dist="lognormal" if self.relative else "normal",
```

Comment:
I think it should always be "normal" for this class, right? We're returning the mean and stddev of a normal posterior.

Reply:
yes, fixed